daml-foundations/daml-ghc/tests/No_concat_clash.daml

```diff
@@ -0,0 +1,22 @@
+-- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its
+-- affiliates.  All rights reserved.
+
+-- This is a compile-time test.
+
+{-# LANGUAGE NoImplicitPrelude #-}
```

Comment:
Is that pragma actually necessary? I think `import Prelude qualified` should be enough.

Reply:
It turns out that no, it's not necessary. Is that an artifact of the GHC module system?

Reply:
martin wrote privately:
> if you `import Prelude qualified`, it is no longer implicitly imported unqualified.
thanks. good to know. i'll amend the test accordingly!